operators/src/processing/expression/codegen.rs

```diff
@@ -105,11 +113,13 @@ pub enum AstNode {
 impl ToTokens for AstNode {
     fn to_tokens(&self, tokens: &mut TokenStream) {
         let new_tokens = match self {
-            Self::Constant(n) => quote! { #n },
+            Self::Constant(n) => quote! { Some(#n) },
+            Self::NoData => quote! { None },
             Self::Variable(v) => quote! { #v },
             Self::Operation { left, op, right } => {
-                quote! { ( #left #op #right ) }
+                quote! { apply(#left, #right, #op) }
             }
+            // TODO:
```

Comment:
what is there to do?

Reply:
was a leftover: [12efcf2](https://github.com/geo-engine/geoengine/pull/599/commits/12efcf2b127327d2604b47108d51413590a648c5)